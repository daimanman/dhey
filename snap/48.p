{"C":1,"Duration":63670600891,"EndTime":"2018-08-23T14:01:31.2330498+08:00","Err":"","FileId":"4","Method":"POST","N":1,"P":{"info":"TEssskkkkkk颠三倒四ST","name":"DXwwM"},"Payload":"","QH":{},"Remark":"压力测试","ReqNums":1,"StartTime":"2018-08-23T14:01:30.7792371+08:00","Status":1,"TaskId":48,"Type":"FIXED","Url":"http://localhost:9188/go1","Z":""}