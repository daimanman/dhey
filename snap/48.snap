{"AvgTotal":0.4229965,"Fastest":0.4229965,"Slowest":0.4229965,"Average":0.4229965,"Rps":2.364085754846671,"AvgConn":0.0049872,"AvgDNS":0.0049872,"AvgReq":0,"AvgRes":0,"AvgDelay":0.4160152,"ConnMax":0.0049872,"ConnMin":0.0049872,"DnsMax":0.0049872,"DnsMin":0.0049872,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0,"DelayMax":0.4160152,"DelayMin":0.4160152,"Lats":[0.4229965],"ConnLats":[0.0049872],"DnsLats":[0.0049872],"ReqLats":[0],"ResLats":[0],"DelayLats":[0.4160152],"Total":422996500,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":51,"SizeReq":51,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.4229965,"Count":1,"Frequentcy":1},{"Mark":0.4229965,"Count":0,"Frequentcy":0},{"Mark":0.4229965,"Count":0,"Frequentcy":0},{"Mark":0.4229965,"Count":0,"Frequentcy":0},{"Mark":0.4229965,"Count":0,"Frequentcy":0},{"Mark":0.4229965,"Count":0,"Frequentcy":0},{"Mark":0.4229965,"Count":0,"Frequentcy":0},{"Mark":0.4229965,"Count":0,"Frequentcy":0},{"Mark":0.4229965,"Count":0,"Frequentcy":0},{"Mark":0.4229965,"Count":0,"Frequentcy":0},{"Mark":0.4229965,"Count":0,"Frequentcy":0}],"TaskId":48,"Duration":0.4229965}